### PR TITLE
fix(acccessconsoles): use object type Select options values in order to not mix displayed text with the option keys

### PR DESCRIPTION
**Pull request by KKoukiou**
Commit 7be4bc4a92e208073ca3e38415df1f64a89506ee introduced a bug where
the displayed SelectOption label is the key and not the expected text
after changing option.

### CODE REVIEW

**Comment by pcbailey** [comment]
Looks good to me! =)

**Comment by KKoukiou** [comment]
Friendy ping - could I get from someone a second review here?

**Review by nicolethoen — APPROVED** [review]

**Review by tlabaj — APPROVED** [review]

**Pull request merged**
